server-spi-private/src/main/java/org/keycloak/authentication/DisplayUtils.java

```diff
@@ -0,0 +1,20 @@
+package org.keycloak.authentication;
+
+import org.keycloak.OAuth2Constants;
+
+/**
+ * Determine OIDC display parameter type.
+ *
+ */
+public class DisplayUtils {
```

Comment:
Wouldn't it be better to add a method directly to AuthenticationFlowContext and RequiredActionContext?

Reply:
Yeah, I'm going to change this.  See my comments way down below.